docs/management.md

```diff
@@ -48,16 +48,22 @@ This guide covers:
  * How to [enable HTTPS for management UI](#single-listener-https) and its underlying API
  * How this plugin [operates in multi-node clusters](#clustering)
  * How to [disable metric collection](#disable-stats) to use [Prometheus](./prometheus) exclusively for monitoring
- * [OAuth 2](#oauth2-authentication) support
+ * [OAuth 2](#oauth2-authentication) support:
```

Comment:
Why not call this what it is: "Authenticating with OAuth 2" here at the top of the page?